framework/src/modules/chain/rounds/round.js

```diff
@@ -300,26 +299,33 @@ class Round {
 		const roundRewards = [...this.scope.roundRewards] || [];
 
 		// Apply exception for round if required
-		if (exceptions.rounds[this.scope.round.toString()]) {
+		if (this.scope.exceptions.rounds[this.scope.round.toString()]) {
```

Comment:
Why do we keep `this.scope` in this file only?

Reply:
I was trying to be minimally invasive for round module (that's why it still has `library` etc)

Reply:
Can we create an issue to make the refactor in separate PR?

Reply:
This is re-written in BFT branch